src/Ybus.jl

```diff
@@ -854,7 +859,7 @@ function Ybus(
             @warn "More than one island found; Network is not connected"
         end
     else
-        subnetworks = Dict{Int, Set{Int}}(bus_lookup[1] => Set(bus_lookup))
+        subnetworks = Dict{Int, Set{Int}}(bus_lookup[1] => Set(keys(bus_lookup)))
```

Comment:
there is something off here. Why keys?

Reply:
Changes this to use the bus_axis not the keys of the lookup